src/main/java/com/sonymobile/jenkins/plugins/lenientshutdown/PluginImpl.java

```diff
@@ -151,10 +169,8 @@ public void run() {
     public boolean isAnyPermittedUpstreamQueueId(Set<Long> queueItemsIds, String nodeName) {
         boolean isPermitted = false;
         Set<Long> permittedQueueItemIds = getPermittedQueuedItemIds(nodeName);
-        if (permittedQueueItemIds != null) {
-            Collection<?> intersection = CollectionUtils.intersection(queueItemsIds, permittedQueueItemIds);
-            isPermitted = !intersection.isEmpty();
-        }
+        Collection<?> intersection = CollectionUtils.intersection(queueItemsIds, permittedQueueItemIds);
```

Comment:
Is this where spotbugs found a "`permittedQueueItemIds` is never `null`" type thing? I get ants under my skin when I see precautions removed like this :)